xds/internal/balancer/clusterimpl/balancer_test.go

```diff
@@ -820,6 +822,108 @@ func (s) TestUpdateLRSServer(t *testing.T) {
 	}
 }
 
+// Test verifies that child policies was updated on receipt of
+// configuration update.
+func (s) TestChildPolicyUpdatedOnConfigUpdate(t *testing.T) {
+	xdsC := fakeclient.NewClient()
+
+	builder := balancer.Get(Name)
+	cc := testutils.NewBalancerClientConn(t)
+	b := builder.Build(cc, balancer.BuildOptions{})
+	defer b.Close()
+
+	// Keep track of which child policy was updated
```

Comment:
Here and elsewhere in this PR, please terminate comment lines with a period. See: go/go-style/decisions#comment-sentences

Reply:
Nit: the distinction is complete sentences and not comments.  Comments _can_ be sentence fragments if it makes sense, in which case we don't use periods.

Reply:
Updated, thAnks.